### PR TITLE
Pin isort version, bump black

### PR DESCRIPTION
**Pull request by mryab**
This PR bumps the version of black to the first [officially stable](https://black.readthedocs.io/en/stable/change_log.html#id1) version of 22.1.0, applying the required formatting changes. Also, it fixes the version of isort to prevent any unwanted inconsistencies in the future.

### CODE REVIEW

**Review by yhn112 — APPROVED** [review]
LGTM

File: tests/test_p2p_daemon_bindings.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ async def test_read_write_unsigned_varint_max_bits_edge(max_bits):
     Test edge cases with different `max_bits`
     """
     for i in range(-3, 0):
-        integer = i + (2 ** max_bits)
+        integer = i + (2**max_bits)
```
Comment (borzunov) [inline tests/test_p2p_daemon_bindings.py:128]:
```suggestion
        integer = i + 2**max_bits
```

File: tests/test_p2p_daemon_bindings.py [hunk 2]
```diff
@@ -94,7 +94,7 @@ async def test_write_unsigned_varint_overflow(integer):
         await write_unsigned_varint(s, integer)
 
 
-@pytest.mark.parametrize("integer", (-1, -(2 ** 32), -(2 ** 64), -(2 ** 128)))
+@pytest.mark.parametrize("integer", (-1, -(2**32), -(2**64), -(2**128)))
```
Comment (borzunov) [inline tests/test_p2p_daemon_bindings.py:97]:
```suggestion
@pytest.mark.parametrize("integer", (-1, -2**32, -2**64, -2**128))
```

File: hivemind/moe/client/switch_moe.py [hunk 3]
```diff
@@ -150,7 +150,7 @@ def forward(self, input: torch.Tensor, *args: torch.Tensor, **kwargs: torch.Tens
         # for each grid dimension, sum across all indices for a dimension. Optimizing this leads to uniform allocation
         balancing_loss = torch.stack(
             [
-                torch.mean(dim_softmax.mean(0) * dim_utilization) * (dim_size ** 2)
+                torch.mean(dim_softmax.mean(0) * dim_utilization) * (dim_size**2)
```
Comment (borzunov) [inline hivemind/moe/client/switch_moe.py:153]:
```suggestion
                torch.mean(dim_softmax.mean(0) * dim_utilization) * dim_size**2
```

File: .github/workflows/check-style.yml [hunk 4]
```diff
@@ -22,3 +22,6 @@ jobs:
         with:
           python-version: 3.8
       - uses: isort/isort-action@master
+        with:
+          isortVersion: "5.10.1"
+
```
Comment (borzunov) [inline .github/workflows/check-style.yml:27]: nit: Too many newlines

**Review by borzunov — APPROVED** [review]
Thanks for the PR!

I think it's worth to clarify that the new release is [the first non-beta release](https://github.com/psf/black/releases/tag/22.1.0) of black. Projects like Django committed to move to black as soon as this release comes out, and I agree we can also do that.

File: tests/test_p2p_daemon_bindings.py (hunk 2 above)
Comment (yhn112) [reply tests/test_p2p_daemon_bindings.py:97]: Disagree
Parenthesis save a few seconds of understanding in cases like this

File: .github/workflows/check-style.yml (hunk 4 above)
Comment (mryab) [reply .github/workflows/check-style.yml]: Removed

File: tests/test_p2p_daemon_bindings.py (hunk 2 above)
Comment (mryab) [reply tests/test_p2p_daemon_bindings.py:97]: I'm also for keeping the brackets in this case
